tests/test_write_json.py

```diff
@@ -0,0 +1,650 @@
+"""test_write_json.py"""
+__author__ = "Marius Pahl"
+
+import json
+import tempfile
+import unittest
+from collections import OrderedDict
+
+import numpy as np
+import pandas as pd
+
+from collect_stata.write_json import (
+    generate_stat,
+    stat_dict,
+    stats_cat,
+    stats_number,
+    stats_string,
+    uni,
+    uni_cat,
+    uni_statistics,
+    write_json,
+)
+
+
+class TestWriteJson(unittest.TestCase):
+    """
+    Tests for all methods from write_json.py
+    """
+
+    def setUp(self):
+        """
+        Set a TemporaryDirectory for the json testfile
+        """
+        self.sandbox = tempfile.TemporaryDirectory()
+        return super().setUp()
+
+    @staticmethod
```

Comment:
Please don't create your static test data in your code like this.
Store it in a json file or in another file format that supports the structure of your data and then load it from there.